src/Bridge/Twig/Extension/EnumExtension.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Greg0ire\Enum\Bridge\Twig\Extension;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class EnumExtension extends \Twig_Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return [
+            new \Twig_SimpleFunction('enum_label', [$this, 'label']),
+        ];
+    }
+
+    /**
```

Comment:
Yes! Useful to show a nice representation of the value and can be combine with translations, like the form type does.

But I still don't know what is the best: A function or a filter? Maybe both? Not hard to do.